app/assets/javascripts/tests/_draw_session.test.js

```diff
@@ -31,8 +31,40 @@ test('drawMobileSessionStartPoint calls drawMarker', t => {
   t.end();
 });
 
-const _drawSession = ({ map }) => {
+test('drawFixedSession creates a marker if heat level is not null', t => {
```

Comment:
Seems like you could move some noise in the builder (`drawSessionMock`) cause it's a bit noisy here